src/lib/linux-api/src/time.rs

```diff
@@ -118,6 +118,52 @@ pub fn alarm(secs: u32) -> Result<u32, Errno> {
     Ok(res.try_into().unwrap())
 }
 
+/// Make a `getitimer` syscall.
+///
+/// # Safety
+///
+/// `curr_value` must be safe for the kernel to write to.
+pub unsafe fn getitimer_raw(which: i32, curr_value: *mut itimerval) -> Result<(), Errno> {
+    unsafe { syscall!(linux_syscall::SYS_getitimer, which, curr_value) }
```

Comment:
Same comment about `__kernel_old_itimerval` and `int`.

Reply:
Done